src/mplfinance/plotting.py

```diff
@@ -787,22 +775,29 @@ def _addplot_columns(panid,panels,ydata,apdict,xdates,config):
         mark  = apdict['marker']
         color = apdict['color']
         alpha = apdict['alpha']
+        labels = apdict["labels"]
```

Comment:
Use single quotes, `apdict['labels']` instead of `apdict["labels"]`, _only for the sake of being consistent with existing/surrounding code style_.

**This is a good, general practice**:  Whenever modifying any existing code, keep to the style of the existing code (assuming that code has a consistent style to begin with).  (If existing style is already inconsistent, then consider modifying the code to be consistent).